monopod/pkg/commands/scaffold.go

```diff
@@ -91,192 +67,70 @@ func Scaffold() *cobra.Command {
 
 // runScaffold does the real work of creating the folders, files, and evaluating
 // templates.
-func (o scaffoldOptions) runScaffold() error {
+func (o scaffoldOptions) runScaffold(ctx context.Context) error {
```

Comment:
```suggestion
func (o scaffoldOptions) runScaffold(_ context.Context) error {
```

Reply:
I think I'd prefer to keep it called `ctx`, so it's clearer that someone doesn't need `context.Background()`